app/models/miq_event_definition_set.rb

```diff
@@ -27,4 +27,6 @@ def self.fixture_path
   def self.display_name(number = 1)
     n_('Event Definition Set', 'Event Definition Sets', number)
   end
+
+  alias_method :events, :miq_event_definitions
```

Comment:
These aren't events though, and it's going to get confusing...they are `event_definitions`.

I'm fine with the alias for now, but I'm wondering if it might be better to change this to no longer have the miq prefix (which should clear things up a lot).  Granted that's a much larger change, hence why I'm fine with an alias.

@kbrock @d-m-u Opinions here?  Especially since I think you've been doing work to untangle sets and make them less magical?

Reply:
Right, these are `event_definitions`. 

The method `events` is defined [here in API](https://github.com/ManageIQ/manageiq-api/blob/6096543dcfce5dd1ca1f6a8582cb7c834678a877/app/controllers/api/subcollections/events.rb#L5) and [here in policy](https://github.com/ManageIQ/manageiq/blob/7b7fd98d3dee0deefaabedb997f3e0088d98982d/app/models/miq_policy.rb#L110).